app/assets/stylesheets/expenses.css.scss

```diff
@@ -2,11 +2,16 @@
   float: right;
 }
 
-#create_expense_link_text {
+#expenses_page_heading {
```

Comment:
Avoid using id selectors<br>Selector `expenses_page_heading` should be written in lowercase with hyphens